ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -1239,27 +1239,15 @@ periodics:
           - as: e2e
             steps:
               cluster_profile: packet
-              env:
-                DEVSCRIPTS_CONFIG: |
-                  IP_STACK=v4
-                  NETWORK_TYPE=OpenShiftSDN
-              workflow: baremetalds-e2e
+              workflow: baremetalds-e2e-ipv4
           - as: e2e-dualstack
```

Comment:
ci-chat-bot doesn't have `dualstack` support iiuc, lets remove this test

Reply:
From metal ipi perspective, dualstack is one of important environment to test. Why do you think ci-chat-bot does not support it?

As far as I know, `test e2e-dualstack 4.10 metal` works.

Reply:
Ah, okay, I wasn't aware. In any case, it should be a different PR to isolate the change